### PR TITLE
Convert SIGNALS and SLOTS to Qt5 signals and slots

### PR DESCRIPTION
**Pull request by sithlord48**
- Stop tracking the string locations in ts files (reduce unneeded updates on them)
- Use Qt5 connections 
  - *ListWidget::addAction needs updating due to how its creates actions*
- Use QPushButton::clicked in place of QPuchButton::released
- Release both Qt5 and Qt6 assets
- Fix up various Clazy / Clang issues

### CODE REVIEW

**Comment by myst6re** [comment]
>     * Release both Qt5 and Qt6 assets

No need to add this, I dropped Qt5 support

File: src/core/field/BackgroundTilesIO.cpp [hunk 1]
```diff
@@ -760,7 +760,9 @@ bool BackgroundTilesIOPS::writeData(const BackgroundTiles &tiles) const
 			}
 
 			// Separation between Y (only when several lines)
-			if (tilesByDstY.keys().size() > 1) {
+			auto tileKeys = tilesByDstY.keys();
+			int tileSize = tileKeys.size();
```
Comment (myst6re) [inline src/core/field/BackgroundTilesIO.cpp:764]: qsizetype instead of int

File: src/core/field/Opcode.cpp [hunk 2]
```diff
@@ -263,7 +263,7 @@ bool Opcode::searchVar(quint8 bank, quint16 address, Operation operation, int va
 			}
 		} else {
 			// Every write vars
-			for (const FF7Var &var : vars) {
+			for (const FF7Var &var : qAsConst(vars)) {
```
Comment (myst6re) [inline src/core/field/Opcode.cpp:266]: already done since var is const

**Comment by sithlord48** [comment]
> > ```
> > * Release both Qt5 and Qt6 assets
> > ```
> 
> No need to add this, I dropped Qt5 support

The CI Still has both ShouldQt5 be removed from the CI ?

File: src/core/field/Opcode.cpp (hunk 2 above)
Comment (sithlord48) [reply src/core/field/Opcode.cpp:266]: Line 227  ```QList<FF7Var> vars;```
This is one is not a const

Comment (myst6re) [reply src/core/field/Opcode.cpp:266]: Yeah, but there no need to set vars as const if var is not modified

**Comment by sithlord48** [comment]
This is done to prevent possible detachment In vars

File: src/core/field/Opcode.cpp (hunk 2 above)
Comment (sithlord48) [reply src/core/field/Opcode.cpp:266]: Vars is set as a const to prevent possible detachment during the loop.

File: src/Window.cpp [hunk 3]
```diff
@@ -1675,7 +1675,7 @@ void Window::miscOperations()
 		hideProgression();
 
 		if (fieldArchive->isModified()) {
-			setModified();
+		setModified(true);
```
Comment (myst6re) [inline src/Window.cpp:1678]: Indentation is missing

**Review by myst6re — CHANGES_REQUESTED** [review]

**Review by myst6re — APPROVED** [review]

**Pull request merged**